src/App.js

```diff
@@ -939,6 +939,9 @@ function HeroPage({ id, data, update }) {
           <span>Level {hero.level}</span>
           <span>Max loyalty:
       {atLevelScaled(heroMeta.max_loyalty_base, heroMeta.max_loyalty_per_level)}
+      </span>
+          <span>Weight:
+      {Math.round(50 * heroMeta.weight)}
```

Comment:
Does it have any effect on the game logic? If not, we shouldn't display it here. Instead let's display the attack power and stuff like that.

Reply:
We already display the attack power (called influence).

No weight does not have a role in the game logic, but it does on the animation. And it's also a fun fact about the hero. What's wrong with displaying it?